agent/src/beerocks/slave/backhaul_manager/backhaul_manager_thread.cpp

```diff
@@ -1739,7 +1739,7 @@ bool backhaul_manager::handle_1905_1_message(ieee1905_1::CmduMessageRx &cmdu_rx,
         return false;
     }
     case ieee1905_1::eMessageType::TOPOLOGY_QUERY_MESSAGE: {
-        return handle_1905_discovery_query(cmdu_rx, src_mac);
+        return handle_1905_topology_query(cmdu_rx, src_mac);
```

Comment:
Excellent that you did this in a separate commit! :1st_place_medal:

Reply:
Thanks